docs/master/testing/phpunit.md

```diff
@@ -140,6 +158,62 @@ public function testOrdersUsersByName(): void
 }
 ```
 
+### Subscription Assertions
+
+#### Testing Authorization
+
+Once you do your subscription query and get a `$response`, you can run the following assertions on it.
+
+If you want to make sure the current user is authorized to join a subscription you can use `assertGraphQLSubscriptionAuthorized`
+
+```php
+$response->assertGraphQLSubscriptionAuthorized($this);
+```
+
+If you want to make sure the current user is NOT authorized to join a subscription you can use `assertGraphQLSubscriptionNotAuthorized`
+
+```php
+$response->assertGraphQLSubscriptionNotAuthorized($this);
+```
+
+#### Testing Broadcast
+
+Once you do your subscription query and get a `$response`, you can run the following assertions on it.
+
+For asserting a broadcast you can use `assertGraphQLBroadcasted`
+
+```
+// any other way to broadcast would laso work
+Subscription::broadcast('postUpdated', ['title' => 'foo']);
+Subscription::broadcast('postUpdated', ['title' => 'bar']);
```

Comment:
Can this be done after the call to the subscription has been made?

Reply:
Yes exactly, first something like this should be called 

```php
public function testPostsSubscription(): void
{
    $response = $this->graphQL(/** @lang GraphQL */ '
    {
        subscription OnPostCreated {
            onPostCreated {
                title
            }
        }
    }
    ');
}
```

Then `Subscription::broadcast` and then `assertGraphQLBroadcasted`